### PR TITLE
Update docs; Mention starknet-devnet-js

### PR DESCRIPTION
**Pull request by FabijanC**
## Usage related changes

What the title says.

## Development related changes

None

## Checklist:

- [x] Checked out the [contribution guidelines](CONTRIBUTING.md)
- [x] Applied formatting - `./scripts/format.sh`
- [x] No linter errors - `./scripts/clippy_check.sh`
- [x] No unused dependencies - `./scripts/check_unused_deps.sh`
- [x] No spelling errors - `./scripts/check_spelling.sh`
- [x] Performed code self-review
- [x] Rebased to the latest commit of the target branch (or merged it into my branch)
- [x] Updated the docs if needed - `./website/README.md`
- [x] Linked the [issues](https://github.com/0xSpaceShard/starknet-devnet-rs/issues) resolvable by this PR - [linking info](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
- [x] Updated the tests if needed; all passing - [execution info](https://github.com/0xSpaceShard/starknet-devnet-rs/?tab=readme-ov-file#test-execution)

### CODE REVIEW

File: website/docs/api.md [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
+Unlike Pythonic Devnet, which also supported Starknet's gateway and feeder gateway API, Devnet in Rust supports [Starknet's JSON-RPC API](https://github.com/starkware-libs/starknet-specs/tree/master/api). Since JSON-RPC v0.6.0, to find out which JSON-RPC version is supported by which Devnet version, check out the [releases page](https://github.com/0xspaceshard/starknet-devnet-rs/releases).
 
-Devnet has many other functional features which are available via their own endpoints and JSON-RPC, which are all mentioned throughout the documentation.
+### Devnet API
+
+Devnet has many additional features which are available via their own endpoints and JSON-RPC, which are all mentioned throughout the documentation.
+
+## Interacting with Devnet in JavaScript and TypeScript
+
+To interact with Devnet usings the [Devnet API](#devnet-api), you can use [`starknet-devnet-js`](https://github.com/0xSpaceShard/starknet-devnet-rs/). This can be especially useful in achieving [L1-L2 communication](./postman.md#l1-l2-interaction-via-postman).
```
Comment (mikiw) [inline website/docs/api.md:17]: [`starknet-devnet-js`](https://github.com/0xSpaceShard/starknet-devnet-rs/)

wrong link -> https://github.com/0xSpaceShard/starknet-devnet-rs/

File: website/docs/blocks.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Devnet starts with a genesis block (with a block number equal to 0). In forking mode, the genesis block number will be equal to the forked block number plus one.
 
-A new block is generated based on the pending block, once a new block is generated the pending block is restarted. By default, a new block is generated with each new transaction, but you can also [create an empty block by yourself](#create-an-empty-block).
```
Comment (mikiw) [inline website/docs/blocks.md:5]: why this was removed "A new block is generated based on the pending block, once a new block is generated the pending block is restarted." wrong resolved conflict?

**Review by mikiw — CHANGES_REQUESTED** [review]

**Review by ivpavici — APPROVED** [review]
lgtm!

File: website/docs/api.md (hunk 1 above)
Comment (FabijanC) [reply website/docs/api.md]: Fixed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/515/commits/1873c579df9e1f520582dc788d67911c556c91be

File: website/docs/blocks.md (hunk 2 above)
Comment (FabijanC) [reply website/docs/blocks.md:5]: I see you resolved, but I felt the need to improve blocks.md: https://github.com/0xSpaceShard/starknet-devnet-rs/pull/515/commits/1873c579df9e1f520582dc788d67911c556c91be

**Review by mikiw — APPROVED** [review]

**Pull request merged**